test/integration/freetext_editor_spec.js

```diff
@@ -45,17 +47,58 @@ const copyPaste = async page => {
   await promise;
 };
 
-const clearAll = async page => {
+const selectAll = async page => {
   await page.keyboard.down("Control");
   await page.keyboard.press("a");
   await page.keyboard.up("Control");
-  await page.waitForTimeout(10);
+  await page.waitForFunction(
+    () => !document.querySelector(".freeTextEditor:not(.selectedEditor)")
+  );
+};
+
+const clearAll = async page => {
+  await selectAll(page);
   await page.keyboard.down("Control");
   await page.keyboard.press("Backspace");
   await page.keyboard.up("Control");
-  await page.waitForTimeout(10);
+  await waitForStorageEntries(page, 0);
+};
+
+const switchToFreeText = async page => {
+  await page.click("#editorFreeText");
+  await page.waitForSelector(".annotationEditorLayer.freetextEditing");
 };
 
+const waitForBeingSelected = (page, selector) =>
+  page.waitForFunction(
+    sel => document.querySelector(sel).classList.contains("selectedEditor"),
+    {},
+    selector
+  );
```

Comment:
This looks quite similar to the general `waitForSelectedEditor` helper, in the `test_utils.js` file; can we perhaps use that one instead?